.storybook/webpack.config.js

```diff
@@ -0,0 +1,61 @@
+const path = require('path');
+const I18nAggregatorPlugin = require('terra-i18n-plugin');
+const i18nSupportedLocales = require('terra-i18n/lib/i18nSupportedLocales');
+const CustomProperties = require('postcss-custom-properties');
+const rtl = require('postcss-rtl');
+
+const customProperties = CustomProperties({ preserve: true, warnings: false });
+customProperties.setVariables({
```

Comment:
is this duplicated from our other webpack?

Reply:
it's copied from terra-consumer-site webpack config. Probably it's not necessary, will remove it and test the code